src/components/TaskPreviewPanel.js

```diff
@@ -1,68 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
```

Comment:
Since _TaskPreviewPanel_ is a functional component, we should stick to the convention and use hook _useTranslations_ instead of _injectIntl_.
```
import { useTranslations } from '@openimis/fe-core';
const { formatMessage } = useTranslations('tasksManagement', modulesManager);
```

Reply:
done